api.swagger.yaml

```diff
@@ -16402,6 +16404,7 @@ components:
             - body
             - state
             - site_section
+            - owner
```

Comment:
This is configurable, so should not be set to required. See not in swagger "The owner is mandatory if configured to be in Invotra admin."